src/main/webapp/app/admin/cleanup-service/cleanup-service.component.html

```diff
@@ -0,0 +1,76 @@
+                    <th scope="col" jhiTranslate="cleanupService.table.deleteFrom">Delete From</th>
+                    <th scope="col" jhiTranslate="cleanupService.table.deleteTo">Delete To</th>
+                    <th scope="col" jhiTranslate="cleanupService.table.lastExecuted">Last Executed</th>
+                    <th scope="col"></th>
+                </tr>
+            </thead>
+            <tbody>
+                @for (operation of cleanupOperations; track operation) {
+                    <tr>
+                        <td>
+                            {{ 'cleanupService.operation.' + operation.name | artemisTranslate }}
+                        </td>
+                        <td>
+                            @if (operation.name !== 'deleteOrphans') {
+                                <div class="col-md-12">
+                                    <jhi-date-time-picker
+                                        id="deleteFromDate"
+                                        [labelName]="'cleanupService.table.deleteFrom' | artemisTranslate"
+                                        [hideLabelName]="true"
+                                        [(ngModel)]="operation.deleteFrom"
+                                        [shouldDisplayTimeZoneWarning]="false"
+                                    />
+                                </div>
+                            }
+                        </td>
+                        <td>
+                            @if (operation.name !== 'deleteOrphans') {
+                                <div class="col-md-12">
+                                    <jhi-date-time-picker
+                                        id="deleteToDate"
+                                        [labelName]="'cleanupService.table.deleteTo' | artemisTranslate"
+                                        [hideLabelName]="true"
+                                        [(ngModel)]="operation.deleteTo"
+                                        [shouldDisplayTimeZoneWarning]="false"
+                                    />
+                                    @if (!areDatesValid(operation)) {
+                                        <span class="invalid-feedback" jhiTranslate="entity.startDateError"></span>
+                                    }
+                                </div>
+                            }
+                        </td>
+                        <td>
+                            @if (operation.lastExecuted === undefined) {
+                                {{ 'cleanupService.notRunYet' | artemisTranslate }}
+                            } @else {
+                                {{ operation.lastExecuted | artemisDate }}
+                            }
+                        </td>
+                        <td>
+                            <button
+                                id="execute-operation"
+                                jhiDeleteButton
+                                [disabled]="!areDatesValid(operation)"
```

Comment:
We should avoid this function call in the template. You might be able to use a signal instead

Reply:
done